libfuse/special_xattr.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libfuse
+
+import (
+	"bytes"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"time"
+)
+
+// finderInfoExtendedAttribute is for extended attributes for the Finder
```

Comment:
why embed into the executable rather than having this as an external file like the volume icon? My main worry is that the gzipped bytes aren't that easy to inspect and verify, and a naive reader of the code might think something nefarious is going on.